assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -148,10 +148,6 @@ const ScheduleDirection = ({
         if (result.length === 0 || current.typicality < result[0].typicality)
           return [current];
         if (current.typicality === result[0].typicality) {
-          if (result[0].shape_priority < 0 && current.shape_priority > 0)
```

Comment:
I'm not sure why this is being removed in the context of this PR. A note in the PR would help.

Reply:
Thanks, totally forgot to mention it. Priority was an attribute of V3 API shapes that's been removed in more recent versions, and as far as I could tell we had already stopped reading/parsing it in our own codebase. Added a note to the main PR text too.